src/components/StartPage.vue

```diff
@@ -1,172 +1,70 @@
 <template>
-  <q-icon
-    :name="$q.dark.isActive ? 'fas fa-sun' : 'fas fa-moon'"
-    :class="[
-      $q.dark.isActive ? 'text-white' : 'text-black',
-      'cursor-pointer fixed-top-right q-mx-xl q-my-lg text-h5',
-    ]"
-    @click="toggleDarkMode"
-  />
-  <div
-    class="q-mx-auto q-my-xl"
-    style="max-width: 600px"
-  >
-    <h5 class="text-h4 q-mb-sm">
-      NER Text Annotator
-    </h5>
-    <p :class="['text-subtitle1', $q.dark.isActive ? 'text-grey-4' : 'text-grey-7']">
+  <div class="q-mx-auto q-my-xl" style="max-width: 600px">
+    <h1 class="text-h4 q-mb-sm text-center">NER Text Annotator</h1>
+    <p class="text-subtitle1 text-center">
       Annotate text for spaCy NER Model training
     </p>
 
     <div class="q-my-xl q-py-md column">
-      <div class="row justify-between items-center">
-        <div>
-          <p :class="['text-h6 q-my-md', $q.dark.isActive ? 'text-grey-4' : 'text-grey-7']">
-            Select your file(s), then press 'Begin'!
-          </p>
-          <p :class="['text-subtitle1 q-my-md', $q.dark.isActive ? 'text-grey-4' : 'text-grey-7']">
-            Hint: You can also drag and drop files into this window!
+      <div class="row justify-around q-my-xl">
+        <div class="col-5 q-mx-sm">
+          <q-file
+            v-model="textFile"
+            accept=".txt"
+            filled
+            label="Load a text file"
+            :color="highlightTextFileInput ? 'red-8' : 'primary'"
+            :bg-color="highlightTextFileInput ? 'red-1' : 'grey-3'"
```

Comment:
You may need to find an alternative way to do this. I definitely think the reddish color for errors improves the UX, but the text is unfortunately unreadable on the dark theme with the background set to `grey-3`.

Interestingly, using `black-1` instead of `grey-3` here results in a darker grey when using the dark theme and a lighter grey when using the light theme. No idea why it behaves that way but it works!